src/proto/Binance.proto

```diff
@@ -75,6 +75,40 @@ message TokenUnfreezeOrder {
     int64 amount = 3;  // amount of token to unfreeze
 }
 
+message HTLTOrder {
+    // 0xB33F9A24      // prefix
+    bytes from = 1;    // signer address
+    bytes to = 2;    // recipient address
+    string recipientOtherChain = 3;
+    string senderOtherChain = 4;
+    bytes randomNumberHash = 5;
+    int64 timestamp = 6;
+    repeated SendOrder.Token amount = 7;
+    string expectedIncome = 8;
+    int64 heightSpan = 9;
+    bool crossChain = 10;
+}
+
+message DepositHTLTOrder {
+    // 0xB33F9A24      // prefix
```

Comment:
what is this prefix used for?

Reply:
https://github.com/trustwallet/wallet-core/pull/627/files#diff-9b3227fc67c8cc727f5c754a916eac8fR29
It is just a comment